examples/plugin/plugin/tracer.cc

```diff
@@ -51,13 +52,15 @@ class Span final : public trace::Span
 
   bool IsRecording() const noexcept override { return true; }
 
-  Tracer &tracer() const noexcept override { return *tracer_; }
+  trace::SpanContext GetContext() const noexcept override { return span_context_; }
+  //  Tracer &tracer() const noexcept override { return *tracer_; }
```

Comment:
I think this should be deleted instead of commented out. Same for `sdk/src/trace/span.h`